modul/asteroidfield.py

```
@@ -1,50 +1,56 @@
         self.asteroid_count = 5
         self.spawn_interval = 5.0
 
     def spawn(self, radius, position, velocity):
         # Randomly select asteroid type based on weights
+        """TODO: add docstring."""
         asteroid_type = random.choices(
-            list(ASTEROID_TYPE_WEIGHTS.keys()),
-            weights=list(ASTEROID_TYPE_WEIGHTS.values())
+            list(C.ASTEROID_TYPE_WEIGHTS.keys()),
+            weights=list(C.ASTEROID_TYPE_WEIGHTS.values())
         )[0]
```

Ruff (0.14.13)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)